index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <!-- HEADER SECTION-->
+    <header class="header__section">
+        <nav class="navbar__section">
+            <h2 class="navbar__logo">Logo</h2>
+            <i class="navbar__menu fa-solid fa-bars"></i>
+            <div class="navbar__links__container">
+                <a href="#" class="navbar__links">Portfolio</a>
+                <a href="#" class="navbar__links">About</a>
+                <a href="#" class="navbar__links">Contact</a>
+            </div>
```

Comment:
Kindly I suggest you use `ul` and `li` tags instead of a `div` to follow HTML's good semantics and best practices. 😄